doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -199,6 +199,12 @@ Bluetooth libraries and services
 
   * Fixed discovery of empty services.
 
+* :ref:`ble_rpc` library:
+
+  * Added host callback handlers for the ``write`` and ``match`` operations of the CCC descriptor.
+  * Fixed the serialization of the GATT attribute ``write`` callback.
```

Comment:
A lot better now. But still a bit unsure about this one. What is serialized? 
- Is it the ``write`` callback applied to the GATT attribute?
- Or something totally different?

Sorry for not understanding the coding world.

Reply:
Based on discussion in Teams, maybe something like this then:

Fixed the serialization of the ``write`` callback applied to the GATT attribute.

Reply:
Ok, done